### ISSUE TITLE
Whatsapp has a limit number of message 

### ISSUE THREAD

**Issue by ashitadiwan** [akshat-rastogi/whatsapp-chat-migrator#2]
Whatsapp has a limit number of message .
can we limit the number of as per whatsapp

**Comment by akshat-rastogi** [comment]
> Whatsapp has a limit number of message .
> can we limit the number of as per whatsapp

Yes, we can consider this as an improvement but I couldn't really find any article which says there is a daily limit on the number of messages. Could you please add reference if you find any ?

**Comment by ashitadiwan** [comment]
Couldn’t find any recent mention about the limit to number of message sending over whatsapp, but to ensure the app doesn’t consider this as a bot activity , we may add a limit to number of message sent over a period of time
